cdap-docs/admin-manual/build.sh

```diff
@@ -20,7 +20,7 @@ source ../vars
 source ../_common/common-build.sh
 
 DEFAULT_XML="../../cdap-common/src/main/resources/cdap-default.xml"
-DEFAULT_XML_MD5_HASH="dfc51846a2f0cf72962ce89de7b458b7"
+DEFAULT_XML_MD5_HASH="e49c16a4b85482ba14e19fc99cd03798"
```

Comment:
You might want to rebase on top of develop and make sure the hash is still correct.

Reply:
I just verified it. Its still the same